.pre-commit-config.yaml

```diff
@@ -0,0 +1,6 @@
+repos:
+  - repo: https://github.com/psf/black
+    rev: 19.10b0 # Replace by any tag/version: https://github.com/psf/black/tags
+    hooks:
+      - id: black
+        language_version: python3 # Should be a command that runs python3.6+
```

Comment:
This config is related to the commit hooks. That means the git will run `black` when you commit something. But what you need is that when you save anything in vs code, the vs code should run `black` and format the code. 

And I also observed that this `pre-commit-config.yaml` file only works when you install `pre-commit` package. But you have not mentioned it anywhere. You should have added that package to `requirements.txt` and commit it.
So the rule of thumb is always to let your team know what are all things you did for that particular story. In here, I don't know how `.pre-commit-config.yaml` works?

Reply:
Yes, Subbu. You are correct. The `pre-commit-config.yaml` file only works when I install a `pre-commit` package. That was bad. I heard about the `pyproject.toml` file and I check that file, that is the **black** formatted configuration file. I add the `.toml` file to my project.

Reply:
Great 👏 

> And I also observed that this pre-commit-config.yaml file only works when you install pre-commit package. But you have not mentioned it anywhere. You should have added that package to requirements.txt and commit it.

How are you handling this?

Reply:
Whenever a person is using my project that person was confused to use this `.pre-commit-config.yaml` file. So, that's why the _config_.  .toml file to my project.

Reply:
And also have another way to use this `.pre-commit-config` file. That is add the `pre-commit` package to the `requirements.txt `file and commit it.